asn1/CommonLedgerProtocol.asn

```diff
@@ -21,81 +21,53 @@ SideProtocolData ::= SEQUENCE OF SEQUENCE {
   protocolData OCTET STRING
 }
 
-PacketRequest ::= SEQUENCE {
-  packet InterledgerPacket,
+Ack ::= SEQUENCE {
+  ack NULL,
```

Comment:
This protocol can also be used in asymmetric cases where only one side is keeping the state. In such cases, the server may not care about whether the client responds or not to a `Fulfill` request. Setting the `requestId` to `0` basically turns it into a notification.